src/components/Work/Tabs/Metadata.js

```diff
@@ -6,85 +6,95 @@ import { formatDate } from "../../../services/helpers";
 
 const styles = {
```

Comment:
Can we start replacing this inline style CSS (`tabContent`) w/ the `emotion` the styled component pattern?